src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
Took a look at what it's gonna take to do the package name switch. It's a better fit to do that work in a separate PR.